code-challenges/arrayShift/array-shift.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+function insertShiftArray(array, value){
+  const middleIndex = Math.ceil(array.length/2)
+  let finalArray = [];
+  for(let i = 0; i < array.length; i++){
+      if ( i === middleIndex){
+     finalArray.push(value);
+     finalArray.push(array[i]);
+    }else{
+     finalArray.push(array[i]);
```

Comment:
This looks really great, there is a way to do it without doing the .push() which would be great knowledge to have!